src/snap-confine.apparmor.in

```diff
@@ -21,6 +21,8 @@
     /usr/lib/@{multiarch}/libseccomp.so* mr,
     /lib/@{multiarch}/libseccomp.so* mr,
 
+    @{PROC}/@{pid}/auxv r,
+
```

Comment:
I thought we said this was *not* needed. IIRC, the debdiff didn't include it. It's fine if it is needed, I'd just like confirmation that it is.